src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>  
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <a href="#tvimage" class="card__link-image">
+        <img
+          src="./images/imac.jpeg"
+          alt="picture"
+          class="image"
+        >
+      </a>
+
+      <h1 class="card__name">
+        Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+      </h1>
+      <section class="card__code">Код товара: 195434</section>
+
+      <section class="card__stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <section class="card__reviews">Отзывов: 5</section>
+      </section>
+
+      <section class="card__price">
+        <section class="card__price-tag">Цена:</section>
+        <section class="card__amount">69 999 грн</section>
```

Comment:
Use `<p>` instead of `<section>`
Read about semantic tags, and where you should use them